__tests__/app.test.js

```diff
@@ -0,0 +1,38 @@
+const request = require("supertest");
+const db = require("../db/connection");
+const app = require("../app");
+const data = require("../db/data/test-data/index");
+const seed = require("../db/seeds/seed");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("GET /api/topics", () => {
+  it("200: should return an array of topic objects each object should contain slug and description properties", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body;
+        topics.forEach((topic) => {
```

Comment:
We need to check the length of the array before iterating over it. There's a jest matcher for this